extensions/procfs/tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+file(GLOB PROCFS_TESTS  "*.cpp")
+
+SET(PROCFS_TEST_COUNT 0)
+FOREACH(testfile ${PROCFS_TESTS})
+    get_filename_component(testfilename "${testfile}" NAME_WE)
+    add_executable("${testfilename}" "${testfile}")
+    target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/extensions/standard-processors")
+    target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/extensions/procfs")
+    target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/libminifi/test/")
+    target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/extensions/standard-processors")
+    createTests("${testfilename}")
+    target_link_libraries(${testfilename} ${CATCH_MAIN_LIB})
+    target_link_libraries(${testfilename} minifi-procfs)
+    target_link_libraries(${testfilename} minifi-standard-processors)
+    MATH(EXPR PROCFS_TEST_COUNT "${PROCFS_TEST_COUNT}+1")
+    add_test(NAME "${testfilename}" COMMAND "${testfilename}")
+ENDFOREACH()
+file(COPY "${CMAKE_SOURCE_DIR}/extensions/procfs/tests/mockprocfs_t0" DESTINATION "${CMAKE_BINARY_DIR}/extensions/procfs/tests/")
+file(COPY "${CMAKE_SOURCE_DIR}/extensions/procfs/tests/mockprocfs_t1" DESTINATION "${CMAKE_BINARY_DIR}/extensions/procfs/tests/")
```

Comment:
are these test resources? sftp tests and script tests copy such resources to somewhere under `${CMAKE_BINARY_DIR}/bin/`

Reply:
good idea, changed it in https://github.com/apache/nifi-minifi-cpp/pull/1152/commits/6d1985dea3427458200b68e5082df02b15e8d3c9